### DISCUSSION TITLE
How do paste on first input from normal mode?

### DISCUSSION THREAD

**Discussion by manoelpqueiroz** [qutebrowser/qutebrowser#8176] [Q&A] [answered]
One thing I would like to be able to do with Qutebrowser is bind the <kbd>Ctrl+V</kbd> key in normal mode to an action to:

- Find the first input and enter insert mode;
- Paste whatever is present in the system clipboard.

I have managed to achieve the first step by configuring this binding in the config file: `config.bind('<Ctrl-V>', 'hint inputs --first', mode='normal')`, but I'm not sure how to further configure to achieve the second step (nor if it is possible).

Can anyone shed me a light on this matter? Thank you very much in advance!

**Comment by The-Compiler** [comment, marked as answer]
You can separate commands with `;;`, and you can use either `insert-text -- {clipboard}` or `fake-key <ctrl-v>` to paste. It looks like a short delay is needed to actually focus the field (as qutebrowser doesn't have a notation of a command being "done", they are basically fired in parallel).

Thus, something like `hint inputs --first ;; cmd-later 10 insert-text -- {clipboard}` seems to work nicely.
